Scripts/sh/buildAllRelease.sh

```
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2024 gluesniffler <159397573+gluesniffler@users.noreply.github.com>
+# SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#!/usr/bin/env sh
+
+# make sure to start from script dir
+if [ "$(dirname $0)" != "." ]; then
+    cd "$(dirname $0)"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)